frontend/device/remarkable/device.lua

```diff
@@ -112,12 +113,30 @@ end
 
 
 function Remarkable:init()
+    -- Check if we were launched from Oxide launcher
+    -- FIXME this could be changed to check parent process of KOReader instead
```

Comment:
```suggestion
    --- @fixme this could be changed to check parent process of KOReader instead
```

Reply:
If we're happy with checking if Oxide is running instead of checking the parent process, we can just delete this line

Reply:
Given that you wrote it, I would assume there's a practical difference of some sort? In any case, if it's there it should read `--- @fixme`. ;-)

Reply:
Gotcha. Checking the parent process is technically more correct IMO. But it's very unlikely that Oxide will be running while KOReader is running but not launched from Oxide. At least for the end user. It was easier to check if Oxide was running rather than checking the parent process, so that's what I implemented. I suppose the `@fixme` can stay to hint towards potential later improvement.

Reply:
I'm not sure I quite understand why it's more correct though? Of course it's more correct if you talk about being launched from Oxide, but looking from a distance that seems somewhat besides the point. I.e., the issue is that Oxide conflicts and does its own power button handling; is that different if it's just running without being responsible for launching?

Reply:
Hmm good point. If Oxide is running at all, its power-button handling will be active, and KOReader shouldn't try and handle it too.

Reply:
Agreed

Reply:
For more context, on a rM1 power button handling would be active, and the input device would be grabbed so you wouldn't be getting any input events for it anyways. It's only on the rM2 that grabbing doesn't work. So yes, just checking to see if it's running would be enough. Oxide also handles updating the screen when using systemd to suspend the device.